runWebpack.sh

```diff
@@ -67,8 +70,15 @@ gitops_config() {
     return 1
   fi
   echo "INFO: gitops_config - setting APP=$portalApp"
+  logoutInactiveUsers="$(jq -r .global.logout_inactive_users < $manifestFile)"
```

Comment:
I think you already add env var in cloud automation, https://github.com/uc-cdis/cloud-automation/pull/818/files#diff-95b4ad3baa4d2705112b07887f1cd448R82,  so you wont need to parse manifest json here